spec.html

```diff
@@ -38265,6 +38265,7 @@ <h1>Runtime Semantics: PerformPromiseAll ( _iteratorRecord_, _constructor_, _res
             1. Assert: _resultCapability_ is a PromiseCapability Record.
             1. Let _values_ be a new empty List.
             1. Let _remainingElementsCount_ be a new Record { [[Value]]: 1 }.
+            1. Let _promiseResolve_ be ? GetMethod(_constructor_, `"resolve"`).
```

Comment:
The result of `GetMethod` could be a function or `undefined`. In that case where it's `undefined`, do you want an error thrown here or in the `Call(_promiseResolve_ ...)` below?

Reply:
I don't have a preference, but in general I guess it's better to throw earlier rather than later.

Would it make sense to just use a `Get` and do the `IsCallable` check inline instead of using `GetMethod` in this case? Otherwise, we'd do a double check for `undefined` (once in the `GetMethod` call and again in this method).

Reply:
I very much want the former, because otherwise an expression like `(class extends Promise { static resolve(){ throw "NOPE" }}).all(iterable)` will behave differently based solely on the emptiness vs. non-emptiness of `iterable` without regard for the state of iterated values.